srml/balances/README.adoc

```diff
@@ -0,0 +1,90 @@
+# Balances Module
+
+The balances module provides the functionality for handling balances.
+
+## Overview
+
+The balances module provides functions for:
+
+- Getting and setting free balance
+- Retrieving total, reserved, and unreserved balances
+- Repatriating a reserved balance to a beneficiary account that exists
+- Transfering a balance between accounts (when not reserved)
+- Slashing an account balance
+- Account removal
+- Lookup of an index to reclaim an account
+- Increasing or decreasing total stake
+- Setting and removing locks on chains that implement `LockableCurrency`
+
+The dispatchable function `transfer` ensures that the sender has signed the transaction. When using the publicly exposed functions in the implementation, you will need to do these checks in your runtime, as many functions will affect storage without ensuring, for example, that the sender is the signer.
+
+## Public Interface
+
+### Types
+
+- Balance
+- AccountId
+
+### Dispatchable Functions
```

Comment:
What is with the other public functions?

Reply:
I started documenting those, but based off conversations I thought we were not putting those in the README as they may not get maintained. https://hackmd.io/GvSBk00ZQ2u9VuwbjbuN9g?view

Reply:
Resolving as we have put everything in the rustdoc